test/ort.jl

```diff
@@ -0,0 +1,34 @@
+using Test
+using PyCall
+import Ghost: play!, Tape, Input
+import ONNX: ONNXCtx, push_call!, julia2onnx, onnx2julia
+
+
+function ort_run(path, args...)
```

Comment:
Will this work in CI? Don't you need to install onnxruntime or is that done somewhere else?

Reply:
Yes, in GitHub Actions I first install Python and onnxruntime and only then run the tests. A more robust way may be to use Conda.jl or similar to install onnxruntime right from Julia, but I'm not really familiar with that.

Reply:
All right. I suppose that might be inconvenient to people testing locally, but I suppose that so is clobbering their pycall environment.